app/src/modules/settings/routes/policies/collection.vue

```diff
@@ -0,0 +1,262 @@
+						_aggregate: { count: 'user' },
+						_groupBy: ['policy'],
+						_sort: 'policy',
+						_limit: -1,
+					},
+					roles: {
+						_aggregate: { count: 'role' },
+						_groupBy: ['policy'],
+						_sort: 'policy',
+						_limit: -1,
+					},
+				},
+				sort: 'name',
+			},
+		});
+
+		policies.value = response.map((policy) => {
+			let icon = 'badge';
+			if (policy.admin_access) icon = 'verified';
+			if (policy.id === PUBLIC_POLICY_ID) icon = 'public';
+
+			return {
+				...translate(policy),
+				public: policy.id === PUBLIC_POLICY_ID,
+				userCount: policy.users?.[0]?.count.user ?? 0,
+				roleCount: policy.roles?.[0]?.count.role ?? 0,
+				icon,
+			};
+		});
+
+		policies.value.sort((a, b) => {
+			if (a.public && !b.public) return -1;
+			if (!a.public && b.public) return 1;
+			return a.name.localeCompare(b.name);
+		});
+	} catch (error) {
+		unexpectedError(error);
+	} finally {
+		loading.value = false;
+	}
+}
+
+function navigateToPolicy({ item }: { item: Policy }) {
+	router.push(`/settings/policies/${item.id}`);
+}
+</script>
+
+<template>
+	<private-view :title="t('settings_permissions')">
+		<template #headline><v-breadcrumb :items="[{ name: t('settings'), to: '/settings' }]" /></template>
+
+		<template #title-outer:prepend>
+			<v-button class="header-icon" rounded icon exact disabled>
+				<v-icon name="admin_panel_settings" />
+			</v-button>
+		</template>
+
+		<template #actions>
+			<search-input
+				v-if="!loading"
+				v-model="search"
+				:autofocus="policies.length > 25"
+				:placeholder="t('search_policy')"
+				:show-filter="false"
+			/>
+
+			<v-button v-tooltip.bottom="t('create_policy')" rounded icon :to="addNewLink">
+				<v-icon name="add" />
+			</v-button>
+		</template>
+
+		<template #navigation>
+			<settings-navigation />
+		</template>
+
+		<template #sidebar>
+			<sidebar-detail icon="info" :title="t('information')" close>
+				<div v-md="t('page_help_settings_policies_collection')" class="page-description" />
+			</sidebar-detail>
+		</template>
+
+		<div v-if="!search || filteredPolicies.length > 0" class="policies">
+			<v-table
+				v-model:headers="tableHeaders"
+				show-resize
+				:items="filteredPolicies"
+				fixed-header
+				item-key="id"
+				:loading="loading"
+				@click:row="navigateToPolicy"
+			>
+				<template #[`item.icon`]="{ item }">
+					<v-icon class="icon" :name="item.icon" :class="{ public: item.public }" />
+				</template>
+
+				<template #[`item.name`]="{ item }">
+					<v-text-overflow :text="item.name" class="name" :class="{ public: item.public }" :highlight="search" />
+				</template>
+
+				<template #[`item.userCount`]="{ item }">
+					<value-null v-if="item.public" />
+				</template>
+
+				<template #[`item.roleCount`]="{ item }">
+					<value-null v-if="item.public" />
+				</template>
+
+				<template #[`item.description`]="{ item }">
+					<v-text-overflow :text="item.description" class="description" :highlight="search" />
+				</template>
```

Comment:
When searching through a large list of policies im getting the following error where it tries to do things on a `string` that is in fact a `null`.
![firefox_4UEBMacM3u](https://github.com/directus/directus/assets/9389634/ea3747d1-87e5-4dda-a67a-d90a27ea4cf7)

```suggestion
				<template #[`item.name`]="{ item }">
					<v-text-overflow v-if="item.name" :text="item.name" class="name" :class="{ public: item.public }" :highlight="search" />
				</template>

				<template #[`item.userCount`]="{ item }">
					<value-null v-if="item.public" />
				</template>

				<template #[`item.roleCount`]="{ item }">
					<value-null v-if="item.public" />
				</template>

				<template #[`item.description`]="{ item }">
					<v-text-overflow v-if="item.description" :text="item.description" class="description" :highlight="search" />
				</template>
```